unittest/force-styles/CMakeLists.txt

```diff
@@ -73,24 +80,30 @@ if(FFT_SINGLE)
 endif()
 foreach(TEST ${MOL_PAIR_TESTS})
   string(REGEX REPLACE "^.*mol-pair-(.*)\.yaml" "MolPairStyle:\\1" TNAME ${TEST})
+  extract_tags(TEST_TAGS ${TEST})
   add_test(NAME ${TNAME} COMMAND test_pair_style ${TEST} WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
   set_tests_properties(${TNAME} PROPERTIES ENVIRONMENT "LAMMPS_POTENTIALS=${LAMMPS_POTENTIALS_DIR};PYTHONPATH=${TEST_INPUT_FOLDER}:$ENV{PYTHONPATH}")
+  set_tests_properties(${TNAME} PROPERTIES LABELS "${TEST_TAGS}")
```

Comment:
Doesn't `TEST_TAGS` have to be a cmake list instead of a space separated string?

Reply:
`TEST_TAGS` is a CMake list.

Reply:
Oh nice, today I learned `string(REGEX MATCHALL ...)` returns a list, great!